src/catch2/matchers/catch_matchers_range_equals.hpp

```diff
@@ -134,18 +160,44 @@ namespace Catch {
             return { CATCH_FORWARD( range ), std::equal_to<>{} };
         }
 
+        /**
+         * Creates a matcher that checks if all elements in a range are equal
+         * to all elements in an initializer list, in some permutation
+         *
+         * Uses `std::equal_to` to do the comparison
+         */
+        template <typename T>
+        constexpr
+        UnorderedRangeEqualsMatcher<std::initializer_list<T>, std::equal_to<>>
+        UnorderedRangeEquals( std::initializer_list<T> range ) {
```

Comment:
I see that this copies the old overloads, but I wonder why I went with overloads instead of defaulted comparison parameter back then.

Reply:
I wondered the same, but was reluctant to change it. Now that you mention it, I added a commit (before mine) to clean this up: 5328f90520fc. Sorry for force-pushing, but I didn't want to put a commit on top that changes it both for the existing code and the new overloads; the diffs are much cleaner this way.